tests/acceptance/pageObjects/FilesPageElement/sharingDialog.js

```diff
@@ -177,18 +179,34 @@ module.exports = {
         .waitForOutstandingAjaxCalls()
         .waitForElementNotPresent('@saveShareButton')
     },
+    /**
+     * Clicks the button to add a new collaborator
+     */
+    clickCreateShare: function () {
+      return this
+        .useXpath()
+        .waitForElementVisible('@createShareButton')
+        .click('@createShareButton')
+        .waitForElementVisible('@createShareDialog')
```

Comment:
you might want to top up with `.waitForAnimationToFinish` because it takes a while till that thing is at the right position. `waitForElementVisible` might come back too early